doc/infrastructure-and-terraform.md

```diff
@@ -1,7 +1,51 @@
-## Infrastructure and Terraform
+# Infrastructure and Terraform
 
 We use infrastructure as code ([Terraform](https://www.terraform.io/)) to deploy
 and manage resources hosting the app. This is stored in the `terraform`
 directory.
 
-In-depth Documentation: [terraform/README.md](/terraform/README.md)
+Read the [In-depth Terraform documentation](/terraform/README.md)
+
+## Tfvar files
+
+We use `tfvar` files to store the secrets and settings used to manage our
+infrastructure.
+
+Each environment has a corresponding file, `dev.tfvars`, `test.tfvars` and
+`prod.tfvars`.
+
+As the contents of these files is sensitive, we have to handle them carefully
+and store them securely.
+
+These files should never be shared via insecure methods and are stored in the
+Azure Keyvault.
+
+If you need to work with the infrastructure, updating or adding an environment
```

Comment:
This sentence is a bit clunky, I'd replace it with:

If you need to work with the infrastructure, for example updating or adding an environment variable, you will need to fetch the relevant file.